saleor/checkout/base_calculations.py

```diff
@@ -0,0 +1,51 @@
+"""Contain functions which are base for calculating checkout properties.
+
+It's recommended to use functions from calculations.py module to take in account plugin
+manager.
+"""
+
+from typing import TYPE_CHECKING, List
+
+from prices import TaxedMoney
+
+from ..core.taxes import quantize_price, zero_taxed_money
+
+if TYPE_CHECKING:
+    # flake8: noqa
+    from .models import Checkout, CheckoutLine
+    from ..discount.types import DiscountsListType
+
+
+def base_checkout_shipping_price(checkout: "Checkout") -> TaxedMoney:
+    """Return checkout shipping price."""
+    if not checkout.shipping_method or not checkout.is_shipping_required():
+        return zero_taxed_money(checkout.currency)
+
+    shipping_price = checkout.shipping_method.get_total()
+    return quantize_price(
+        TaxedMoney(net=shipping_price, gross=shipping_price), shipping_price.currency
+    )
+
+
+def base_checkout_subtotal(
```

Comment:
This function can be more generic as It doesn't execute any checkout logic. It just sums up a list of taxed money.
`def sum_prices(prices: List[TaxedMoney], currency)`